src/sssom/util.py

```diff
@@ -1358,3 +1358,34 @@ def safe_compress(uri: str, converter: Converter) -> str:
             f"CURIE appeared where there should be a URI, and could not be standardized: {uri}"
         )
     return rv
+
+
+def ensure_valid_mapping_from_dict(mdict):
```

Comment:
Given the recent discussion about people using stuff that should be marked as private, let's move these functions into the `parsers.py` module and keep them marked as private by having an underscore at the beginning of the name

Reply:
Ok done!